From 35d90a5655d4870c51c8359590c86aaa4602be27 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1=C5=A1=20Kelemen=20=28vudiq=29?= <vudiq@vudiq.sk>
Date: Sun, 2 Feb 2025 11:15:33 +0100
Subject: [PATCH] add missing include to fix build with gcc14

without this header file compiler complains about reverse not being a
member of std.
---
 Source/Core/Core/Debugger/PPCDebugInterface.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/Core/Core/Debugger/PPCDebugInterface.cpp b/Source/Core/Core/Debugger/PPCDebugInterface.cpp
index f44c20d7f28c..1ebd0f8d5b48 100644
--- a/Source/Core/Core/Debugger/PPCDebugInterface.cpp
+++ b/Source/Core/Core/Debugger/PPCDebugInterface.cpp
@@ -4,6 +4,7 @@
 
 #include "Core/Debugger/PPCDebugInterface.h"
 
+#include <algorithm>
 #include <array>
 #include <cstddef>
 #include <string>
